README.md

```diff
@@ -157,7 +157,7 @@ cd src
 lrelease QOwnNotes.pro
 
 qmake
-make
+make -j4
```

Comment:
That depends on the amount of cores the system has...

Reply:
Yeah, but [here](https://github.com/pbek/QOwnNotes#building-qownnotes-for-development) it's the same

Reply:
Ok, then :)